api/src/main/webapp/WEB-INF/queue.yaml

```diff
@@ -68,3 +68,15 @@ queue:
 
   retry_parameters:
     task_retry_limit: 0
+
+- name: deleteTestUserWorkspacesQueue
+  target: api
+
+  # rate parameters
+  bucket_size: 50
```

Comment:
Parameters for the [Token Bucket algorithm](https://cloud.google.com/appengine/docs/legacy/standard/python/config/queueref) which defines how tasks are scheduled:
* Running a task "spends" a "token" from a "bucket" (not a GCS bucket)
* This bucket fills up at a **rate** of 1 token per second with a max **bucket_size** of 50.
* As long as tokens are available, 10 tasks can run at a time: **max_concurrent_requests**

I estimate we'll need only 10-ish tasks per day, so this is way more than we need.  We can follow up by restricting this if we need to.